Firestore/Source/Core/FSTQuery.h

```diff
@@ -244,9 +255,19 @@ typedef NS_ENUM(NSInteger, FSTRelationFilterOperator) {
  */
 - (instancetype)queryByAddingEndAt:(FSTBound *)bound;
 
+/**
+ * Helper to convert a Collection Group query into a collection query at a specific path. This is
```

Comment:
Inconsistent case on "Collection Group". elsewhere we call this just "collection group". I don't think these are proper nouns so just lowercase seems fine.

Reply:
done